app/customer_routes.py

```diff
@@ -0,0 +1,243 @@
+from re import U
+from flask.wrappers import Response
+from werkzeug.wrappers import ResponseStreamMixin
+from app import db
+from app.models.customers import Customer
+from app.models.videos import Video
+from app.models.rentals import Rental
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime,timedelta
+import os, requests
+
+customers_bp = Blueprint("cumtomers",__name__,url_prefix="/customers")
+videos_bp = Blueprint("videos",__name__,url_prefix="/videos")
+rentals_bp = Blueprint("rentals",__name__,url_prefix="/rentals")
+
+
+@customers_bp.route("", methods=["POST","GET"], strict_slashes = False)
+def create_customers():
+
+    if request.method == "POST":
+        request_body = request.get_json()
+        
+        if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body: 
+            return ({"details":"Invalid data"},400)
```

Comment:
I suggest creating functions to check for invalid fields in the request body.  You could reuse them in several functions.